model_bakery/baker.py

```diff
@@ -113,10 +117,11 @@ def prepare_recipe(baker_recipe_name, _quantity=None, _save_related=False, **new
 class ModelFinder(object):
     """Encapsulates all the logic for finding a model to Baker."""
 
-    _unique_models = None
-    _ambiguous_models = None
+    def __init__(self) -> None:
+        self._unique_models = None  # type: Optional[Dict[str, Type[Model]]]
+        self._ambiguous_models = None  # type: Optional[List[str]]
```

Comment:
I think this came up with some of the version compatibility work. I did this originally because there wasn't support for setting types on class attributes, and then I needed to do the comment-style attributes anyways. I've fixed it back to the other style now.